app/models.py

```diff
@@ -1438,6 +1438,11 @@ class Notification(db.Model):
 
     postage = db.Column(db.String, nullable=True)
 
+    unsubscribe_link = db.Column(db.String, nullable=True)
+
+    # unsubscribe_link value should be null for non-email notifications
+    CheckConstraint("notification_type = 'email' OR unsubscribe_link is null")
```

Comment:
i think this should be in the `__table_args__` below?

https://docs.sqlalchemy.org/en/14/orm/declarative_tables.html#declarative-table-configuration

might be a good idea to check the other constraints in the file - most are in __table_args__ but there's a couple of stragglers.

i dont think has a huge impact on us (it's more used when sqlalchemy is used to generate db schemas, whereas we just use alembic), but would be nice to get it consistent

Reply:
Yeah, I think the syntax was half-like-this, half-like-that. I don't mind either way, so I put them all in table args variables. Ready for another look ☺️